dags/vds_pull_vdsdata.py

```diff
@@ -132,19 +132,18 @@
         summarize_v15_bylane_task
 
     with TaskGroup(group_id='data_checks') as data_checks:
-        divisions = [2, 8001]
+        divisions = [2, ] #div8001 is never summarized and the query on the view is not optimized
         for divid in divisions:
             check_avg_rows = SQLCheckOperatorWithReturnValue(
                 task_id=f"check_rows_vdsdata_div{divid}",
                 sql="select/select-row_count_lookback.sql",
                 conn_id='vds_bot',
-                params={"table": 'vds.raw_vdsdata',
+                params={"table": f'vds.counts_15min_div{divid}',
```

Comment:
Not specifically for this PR, but I think it's better to use either double or single quotes but not both. Most linters/standards prefer double quotes though